### PR TITLE
feat(rds/instance): add collation support

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:
Only instance create API support collation parameter, List instance response does not contain this attribute, so we don't set it in Read function.

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/rds/ TESTARGS='-run=TestAccRdsInstance_sqlserver'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds/ -v -run=TestAccRdsInstance_sqlserver -timeout 360m -parallel 4
=== RUN   TestAccRdsInstance_sqlserver
=== PAUSE TestAccRdsInstance_sqlserver
=== CONT  TestAccRdsInstance_sqlserver
--- PASS: TestAccRdsInstance_sqlserver (1065.50s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       1065.535s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go [hunk 1]
```diff
@@ -162,6 +162,34 @@ func TestAccRdsInstance_mysql(t *testing.T) {
 	})
 }
 
+func TestAccRdsInstance_sqlserver(t *testing.T) {
+	var instance instances.RdsInstanceResponse
+	name := acceptance.RandomAccResourceName()
+	resourceType := "huaweicloud_rds_instance"
+	resourceName := "huaweicloud_rds_instance.test"
+	pwd := fmt.Sprintf("%s%s%d", acctest.RandString(5), acctest.RandStringFromCharSet(2, "!#%^*"),
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go:170]: please try to use acceptance.RandomPassword instead

File: docs/resources/rds_instance.md [hunk 2]
```diff
@@ -163,6 +163,9 @@ The following arguments are supported:
 * `param_group_id` - (Optional, String, ForceNew) Specifies the parameter group ID. Changing this parameter will create
   a new resource.
 
+* `collation` - (Optional, String, ForceNew) Specifies the Character Set, only available to Microsoft SQL Server DB instances.
```
Comment (ShiChangkuo) [inline docs/resources/rds_instance.md:166]: as we does not set it, should we add this parameter into ignore  changes when importing?

Comment (niuzhenguo) [reply docs/resources/rds_instance.md:166]: sure

**Comment by niuzhenguo** [comment]
make testacc TEST=./huaweicloud/services/acceptance/rds/ TESTARGS='-run=TestAccRdsInstance_sqlserver'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds/ -v -run=TestAccRdsInstance_sqlserver -timeout 360m -parallel 4
=== RUN   TestAccRdsInstance_sqlserver
=== PAUSE TestAccRdsInstance_sqlserver
=== CONT  TestAccRdsInstance_sqlserver
--- PASS: TestAccRdsInstance_sqlserver (1092.83s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       1092.862s

File: huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go (hunk 1 above)
Comment (niuzhenguo) [reply huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go:170]: that wrapped function generated password not satisfied rds's requirements

**Pull request merged**